### PR TITLE
feat(cli, config): `cli.bannerColor` option

### LINKED ISSUE #6397: Change banner color
### What problem does this feature solve?

Perhaps this is the strangest feature request in this repository...<br/>
I think we should update the banner color of `nuxt-ts` to match the color scheme :D

### Nuxt TypeScript

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9760">#c9760</a>)</em></sub></div>

### PR DESCRIPTION
**Pull request by danielroe**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
This exposes a `build.bannerColor` option to change the color of the Nuxt title in the CLI banner.

Resolves: #6397

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes
   *I think that might be overkill for a simple cosmetic change*
- [x] All new and existing tests are passing.

### CODE REVIEW

**Review by pi0 — CHANGES_REQUESTED** [review]
We don't include typescript logic here. It may be better an option to customize color.

**Comment by danielroe** [comment]
> We don't include typescript logic here. It may be better an option to customize color.

I've exposed a `build.ui.titleColor` option, with the idea that more ui options could be exposed in future. Is this the right location for this option? If so, I'll prepare a PR updating `@nuxt/typescript` and `@nuxt/types`.

**Review by galvez — CHANGES_REQUESTED** [review]
I'd call it `bannerColor` instead of `ui.titleColor`, since we don't immediately have any other options to place under that key.

**Comment by danielroe** [comment]
> I'd call it `bannerColor` instead of `ui.titleColor`, since we don't immediately have any other options to place under that key.

Done!

File: packages/cli/src/utils/banner.js [hunk 1]
```diff
@@ -20,7 +20,11 @@ export function showBanner (nuxt, showMemoryUsage = true) {
   const messageLines = []
 
   // Name and version
-  titleLines.push(`${chalk.green.bold('Nuxt.js')} ${nuxt.constructor.version}`)
+  let bannerColor = 'green'
```
Comment (pi0) [inline packages/cli/src/utils/banner.js]: Please keep defaults in config package (test snapshots require update)

**Review by pi0 — CHANGES_REQUESTED** [review]

**Review by pi0 — APPROVED** [review]
LGTM! Thanks ❤️

**Comment by kevinmarrec** [comment]
@danielroe @pi0 Is there any reason the banner border color hasn't be set to the new option ? :P

**Comment by kevinmarrec** [comment]
image

**Comment by Exeteres** [comment]
I have the same question :flushed:

**Comment by danielroe** [comment]
@kevinmarrec @Exeteres I like the idea! (Mind you, in production mode there will be green in the picture too.)

**Pull request merged**